src/streaming/constants/Constants.js

```diff
@@ -228,6 +228,14 @@ export default {
     SUPPLEMENTAL_PROPERTY_DVB_LL_SCHEME: 'urn:dvb:dash:lowlatency:critical:2019',
     THUMBNAILS_SCHEME_ID_URIS: ['http://dashif.org/thumbnail_tile', 'http://dashif.org/guidelines/thumbnail_tile'],
     FONT_DOWNLOAD_DVB_SCHEME: 'urn:dvb:dash:fontdownload:2014',
+    ESSENTIAL_PROP_SCHEME: [
```

Comment:
Thanks, I share that concern.
What about my updated code in `Constants.js` and `Settings.js`? I hope it addresses your concern.

Reply:
This looks good thanks. Just one more comment, I suggest we split up the words in the constants for HDR, e.g. `COLOUR_PRIMARIES_SCHEME_ID_URI` instead of `COLOURPRIMARIES_SCHEME_ID_URI` and so on

Reply:
I think this is ok for now. In my opinion, the next step here is to use the MediaCapabilitiesAPI to check for support on the underlying platform (as discussed in the last dash.js call)